deepdoctection/utils/file_utils.py

```diff
@@ -249,8 +249,23 @@ def get_detectron2_requirement() -> Requirement:
 
 
 # Tesseract related dependencies
-
 _TESS_AVAILABLE = which("tesseract") is not None
+_TESS_PATH = "tesseract"
+
+def set_tesseract_path(tesseract_path: str="tesseract"):
+    global _TESS_AVAILABLE
```

Comment:
Please add a short docstring, e.g. what path to add for installation with Anaconda